android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -70,7 +70,8 @@
 		TiC.PROPERTY_WINDOW_PIXEL_FORMAT,
 		TiC.PROPERTY_FLAG_SECURE,
 		TiC.PROPERTY_BAR_COLOR,
-		TiC.PROPERTY_STATUS_BAR_COLOR
+		TiC.PROPERTY_STATUS_BAR_COLOR,
+		"uiFlags"
```

Comment:
New property?

Reply:
I wasn't sure if I should add one as I only use it in this place. But I can do it to make it be inline with the other values :+1: